tokens/lib/tokens.json

```diff
@@ -1825,6 +1825,20 @@
               "value": "{ghost}",
               "type": "color"
             }
+          },
+          "ai": {
+            "default": {
+              "value": "{color.teal.400}",
```

Comment:
Why do we have teal.400 here and below? It is not used anywhere in the bot today, is it present in the sketches?

Reply:
The only place I used it was the dark theme (TOTT): we have slightly brighter colors here for all interactive elements (brand color is referring to color.blue.400 for instance)

Reply:
Perfect, thanks for clarifying 👍